spec/datadog/core/crashtracking/component_spec.rb

```diff
@@ -318,6 +375,82 @@
           end
         end
       end
+
+      describe 'Ruby and C method runtime stack capture' do
+        let(:runtime_stack) { crash_report_experimental[:runtime_stack] }
+
+        it 'captures both Ruby and C method frames in mixed stacks' do
+          expect_in_fork(fork_expectations: fork_expectations, timeout_seconds: 15) do
+            crash_tracker = build_crashtracker(agent_base_url: agent_base_url)
+            crash_tracker.start
+
+            ComponentSpecCrashStackHelper.new.top_level_ruby_method
```

Comment:
Minor: Since the `ComponentSpecCrashStackHelper` is only needed here, I suggest inlining it here. That is, keep it as-is (that's fine), just move all of the class definition to inside `expect_in_fork`, and then call it. Ruby doesn't mind, I promise ;)